lib/components/narrative/narrative-itineraries.js

```diff
@@ -404,30 +406,37 @@ class NarrativeItineraries extends Component {
               {groupItineraries && !itineraryIsExpanded
                 ? Object.keys(groupedMergedItineraries.multi).map((mode) => {
                     return (
-                      <div key={mode} style={{ padding: '0 1em' }}>
+                      <li key={mode} style={{ padding: '0 1em' }}>
```

Comment:
I don't think the list is needed here, as headings already follow (2 instances).